tests/home_work.spec.js

```diff
@@ -0,0 +1,36 @@
+import { test, expect } from '@playwright/test';
+   
+     test('Perform Login', async ({ page }) => {
+      await page.goto('https://www.saucedemo.com/');
```

Comment:
Please, add baseURL to the playwright config file (you need to find playwright config file, open it and find the commented baseURL property, uncomment it and replace default value with 'https://www.saucedemo.com'. Then in the test if u need to go to the main page u'll do it like `await page.goto('/');`)